pages/linux/w.md

```diff
@@ -0,0 +1,20 @@
+# w
+
+> Display who is logged in and their processes.
+> More information: <https://www.geeksforgeeks.org/w-command-in-linux-with-examples/>.
+
+- Display information of all users logged in:
```

Comment:
Sorry to be picky, but perhaps this will read better?
```suggestion
- Display information about all users who are currently logged in:
```

Reply:
Your suggestion looks more readable. Thanks!